app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
+    command_split = command.split()
+    if len(command_split) != 3 or command[0] != "copy":
+        return
```

Comment:
How should a user understand if they enter an incorrect command?
Please add `print` with some informative  message before `return`